src/Dinkbit/Notifyme/Gateways/Twilio.php

```diff
@@ -140,11 +140,11 @@ protected function parseResponse($body)
      */
     protected function responseError($rawResponse)
     {
-        if (! $this->isJson($rawResponse->getBody())) {
-            return $this->jsonError($rawResponse);
+        if ($json = $this->isJson($rawResponse->getBody())) {
```

Comment:
Done `return $this->isJson($rawResponse->getBody()) ?: $this->jsonError($rawResponse);`